package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-neo",
-  "version": "0.6.2",
+  "version": "0.7.0",
```

Comment:
in other places, we've added engines to this file to specify the npm version -- might be a good idea here too